### PR TITLE
Resolve erblint errors

### PR DESCRIPTION
**Pull request by despo**

### CODE REVIEW

File: app/components/provider_interface/interview_and_course_summary_component.html.erb [hunk 1]
```diff
@@ -5,13 +5,13 @@
         <div class="app-summary-card__actions">
           <ul class="app-summary-card__actions-list">
             <li class="app-summary-card__actions-list-item">
-              <%= link_to edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link govuk-link--no-visited-state' do %>
+              <%= govuk_link_to edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item -link govuk-link--no-visited-state' do %>
```
Comment (cwrw) [inline app/components/provider_interface/interview_and_course_summary_component.html.erb:8]: could you remove the hanging ` -link` here and below

**Review by cwrw — APPROVED** [review]
👍  just a comment around a missed class removal

File: app/components/provider_interface/interview_and_course_summary_component.html.erb [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
+              <%= govuk_link_to edit_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item  govuk-link--no-visited-state' do %>
                 <span class="govuk-visually-hidden"> <%= interview.date_and_time.to_s(:govuk_date_and_time) %> </span>
                 Change details
               <% end %>
             </li>
             <li class="app-summary-card__actions-list-item">
-              <%= link_to cancel_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item govuk-link govuk-link--no-visited-state' do %>
+              <%= govuk_link_to cancel_provider_interface_application_choice_interview_path(application_choice, interview), class: 'app-summary-card__actions-list-item  govuk-link--no-visited-state' do %>
                 <span class="govuk-visually-hidden"> <%= interview.date_and_time.to_s(:govuk_date_and_time) %> </span>
```
Comment (AgaDufrat) [inline app/components/provider_interface/interview_and_course_summary_component.html.erb:15]: I know this is stoping the builds so we should merge it quickly.
But while you are here and it's not too much bother could you also swap the `Change/Cancel interview` with the `date_and_time` so it appears in the same order as in the prototype, as mentioned here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4131#discussion_r581000059

Also happy to do it in a separate PR.

**Review by AgaDufrat — APPROVED** [review]
LGTM

**Pull request merged**